assignments/simple-chat.adoc

```diff
@@ -4,22 +4,21 @@ A simple chat
 Implement a simple chat using `async-std`.
 
 In this exercise, you will learn:
+
 * How to use async tasks for synchronisation
 * How to do simple connection handing
 * How to use async channels
 
-Note: this is a trimmed down version ot the tutorial found in the https://book.async.rs/tutorial/index.html[async-std book].
+Note: this is a trimmed down version of the tutorial found in the https://book.async.rs/tutorial/index.html[async-std book].
 
-1. Clone the template
-~~~~~~~~~~~~~~~~~~~~~
+== 1. Clone the template
 
 [source,rust]
 ----
 $ git clone git@github.com:skade/async-chat-template.git
 ----
 
-2. The protocol
-~~~~~~~~~~~~~~~
+== 2. Protocol Specification
```

Comment:
Let's apply the same transform for the top-level name as well? (should get only one `=`).

Reply:
duh! good point. done.